src/lib.rs

```diff
@@ -907,22 +916,22 @@ struct Key {
 }
 
 impl Key {
-    fn generate() -> Result<(Self, crypto::pkcs8::Document), Error> {
+    pub fn generate() -> Result<(Self, crypto::pkcs8::Document), Error> {
```

Comment:
I'm okay with making `generate()` and `from_pkcs8_der()` public. I don't think `new()` and `to_pkcs8_der()` need to become public.

@cpu thoughts?

Reply:
> @cpu thoughts?

Yeah, in the absence of some additional motivation I agree :+1: